aaronsekisambu/next-web
# Responsive website

## Description
- Working with media queries, grids and flex to style a complete website

## How to Manually Test
- Clone the repo and fetch all branches from the origin
- Check to `develop` branch
- Run the index.html file

## Screenshots
